store/store.go

```diff
@@ -2,16 +2,78 @@ package store
 
 import "context"
 
+type Backend uint8
+
+const (
+	EigenDA Backend = iota
+	Memory
+	S3
+	Redis
+
+	Unknown
+)
```

Comment:
```suggestion
const (
	EigenDABackendType BackendType = iota
	MemoryBackendType
	S3BackendType
	RedisBackendType

	UnknownBackendType
)
```

Reply:
Prefer to be full explicit with naming. When I first saw "EigenDA" being returned from the Backend() function, I assumed it was returning a struct, which didn't make much sense.

Reply:
[265b7b9](https://github.com/Layr-Labs/eigenda-proxy/pull/104/commits/265b7b99a0f235eda7e3e3c4dfb9a2a89a883364)